ballerina/Ballerina.toml

```diff
@@ -10,7 +10,7 @@ artifactId = "jnats"
 version = "@nats.client.version@"
 
 [[platform.java11.dependency]]
-path = "../nats-native/build/libs/nats-native-@project.version@.jar"
+path = "../native/build/libs/nats-native-@project.version@.jar"
 groupId = "ballerina"
 artifactId = "nats"
 version = "@project.version@"
```

Comment:
We can remove these entries since we directly point to the path of jar file. Check the other places as well.

Reply:
f8f7070e1158807641c89ae49d39d7d1c0b3f21c